### PR TITLE
Addresses #3086 NTR pulmonary interlobular septum

### PR DESCRIPTION
**Pull request by aleixpuigb**
Addresses #3086 NTR pulmonary interlobular septum

### CODE REVIEW

**Comment by aleixpuigb** [comment]
#gogoeditdiff

**Review by bvarner-ebi — COMMENTED** [review]
See inline comments.

Will approve if requester and/or SME confirm no changes are desired in response to comments.

File: src/ontology/uberon-edit.obo
Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:224554]: Thank you for your comment. The definition of surrounds is "the inverse of surreounded by", which is:
> x surrounded_by y if and only if (1) x is adjacent to y and for every region r that is adjacent to x, r overlaps y (2) the shared boundary between x and y occupies the majority of the outermost boundary of x

Although the `pulmonary interlobular septum` has some micropores, it occupies the majority of the outermost surface around the `secondary pulmonary lobule`.

From PMID:3259815: "Surrounding the secondary lobule are interlobular septa containing venules and lymphatics"
Supported with the image:
image

I can add this [reference](https://www.researchgate.net/figure/Drawing-of-the-secondary-pulmonary-lobule-of-the-lung-The-interlobular-septa-runs-along_fig2_346517053) from DOI:[10.1016/j.ejro.2020.100274](http://dx.doi.org/10.1016/j.ejro.2020.100274) if it makes it more clear.

File: src/ontology/uberon-edit.obo
Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:224545]: Agree is descriptive, I will remove it.

File: src/ontology/uberon-edit.obo
Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:224552]: - As one reference referred to venules and the other to veins, I have looked for an additional reference that also states that venules go through the septa. I will Add `pulmonary venule` as a new term, and also add this reference. PMID:[22283705](https://pubmed.ncbi.nlm.nih.gov/22283705).

- Which object property would you recommend?

Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:224552]: I will remove the pulmonary vein axiom, and in ticket #3096 I will add:
'pulmonary interlobular septum' SubclassOf overlaps some 'pulmonary venule'.

Comment (bvarner-ebi) [reply src/ontology/uberon-edit.obo]: > Which object property would you recommend?

Not sure if there is a standard in how to relate vessels passing through a structure or if all septa would "overlap" with these vessels. This is not a breaking edit, and defer to SME's guidance.

**Review by bvarner-ebi — APPROVED** [review]

**Pull request merged**
